loopd/swapclient_server.go

```diff
@@ -894,8 +900,9 @@ func validateConfTarget(target, defaultTarget int32) (int32, error) {
 
 	// Ensure the target respects our minimum threshold.
 	case target < minConfTarget:
-		return 0, fmt.Errorf("a confirmation target of at least %v "+
+		log.Errorf("a confirmation target of at least %v "+
```

Comment:
I think it's nicer that a formatted error was returned as clients get more context on the error that way vs monitoring the `loopd` logs.